src/foam/u2/view/RadioButton.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'RadioButton',
+  extends: 'foam.u2.Element',
+
+  documentation: 'A single radio button. Logic implemented in RadioView.js',
+
+  imports: ['theme?'],
+
+  css: `
+  ^innerCircle.selected{
+    r: 5px;
+  }
+  ^innerCircle{
+    transition: 0.1s ease;
+    transform-origin: center center;
+    transform-box: stroke-box;
+    r: 0px;
```

Comment:
You don't need the 'px'

Reply:
Safari doesn't like it if we don't have px